src/panels/config/integrations/integration-panels/zwave_js/dialog-zwave_js-add-node.ts

```diff
@@ -61,42 +93,155 @@ class DialogZWaveJSAddNode extends LitElement {
           this.hass.localize("ui.panel.config.zwave_js.add_node.title")
         )}
       >
-        ${this._status === ""
-          ? html`
-              <p>
-                ${this.hass.localize(
-                  "ui.panel.config.zwave_js.add_node.introduction"
-                )}
-              </p>
-              <div class="secure_inclusion_field">
+        ${this._status === "loading"
+          ? html`<div style="display: flex; justify-content: center;">
+              <ha-circular-progress size="large" active></ha-circular-progress>
+            </div>`
+          : ""}
+        ${this._status === "choose_strategy"
```

Comment:
This can all be chained if you would want.
```suggestion
          : this._status === "choose_strategy"
```